rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/350_point_in_time.yml

```diff
@@ -23,6 +23,10 @@ setup:
   - do:
       indices.create:
         index:  test2
+        body:
+          settings:
+            index:
+              number_of_shards: 2
```

Comment:
I had to change this to be able to reproduce the original test failure, hence I think it's a good idea to not always run this with a single shard. Still trying to figure out what has changed meanwhile in how we run yaml tests that did not let me reproduce the failure anymore.